src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -304,6 +304,26 @@ public FeedVersion retrieveLatest() {
         return newestVersion;
     }
 
+    public FeedVersion retrievePublished() {
```

Comment:
Please add a comment here and also change the name of this method to retrivePublishedVersion.